doc/tutorials/acrn_on_qemu.rst

```diff
@@ -0,0 +1,268 @@
+      --ram 4096 \
+      --disk path=/var/lib/libvirt/images/acrnsos.img,size=32 \
+      --vcpus 4 \
+      --virt-type kvm \
+      --os-type linux \
+      --os-variant ubuntu18.04 \
+      --graphics none \
+      --clock offset=utc,tsc_present=yes,kvmclock_present=no \
+      --qemu-commandline="-machine kernel-irqchip=split -device intel-iommu,intremap=on,caching-mode=on,aw-bits=48" \
+      --location 'http://archive.ubuntu.com/ubuntu/dists/bionic/main/installer-amd64/' \
+      --extra-args "console=tty0 console=ttyS0,115200n8"
+
+2. Walk through the installation steps as prompted. Few things to note,
+
+   a. Make sure to install an open-ssh server so that once the installation is complete, we can ssh into the system.
+
+      .. figure:: images/acrn_qemu_1.png
+         :align: center
+
+   b. We use GRUB to boot ACRN, so please install it when prompted.
+
+      .. figure:: images/acrn_qemu_2.png
+         :align: center
+
+3. To login to the SOS guest, find the IP address of the guest to SSH. This can be done via the
+virsh command as shown below,
+
+   .. figure:: images/acrn_qemu_3.png
+      :align: center
+
+4. Once ACRN hypervisor is enabled, the above virsh command might not display the IP. So enable Serial console by,
+
+   .. code-block:: none
+
+      sudo systemctl enable serial-getty@ttyS0.service
+      sudo systemctl start serial-getty@ttyS0.service
+
+5. Enable grub menu to choose between Ubuntu Vs ACRN hypervisor. Modify :file:`/etc/default/grub` and edit below entries,
+
+      .. code-block:: none
+
+         GRUB_TIMEOUT_STYLE=menu
+         GRUB_TIMEOUT=5
+         GRUB_CMDLINE_LINUX_DEFAULT=""
+         GRUB_GFXMODE=text
+
+6. sudo update-grub
+
+7. Once the above steps are done, SOS guest can also be launched using, ``virsh start ACRNSOS --console``. Make sure to use the domain name
+you used while creating the VM instead of ``ACRNSOS``.
+
+This concludes setting up of SOS and preparing it to boot ACRN hypervisor.
+
+
+Install ACRN Hypervisor
+***********************
+
+1. Clone ACRN hypervisor.
+
+   .. code-block:: none
+
+      $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+      $ cd acrn-hypervisor/
+
+2. Use the following command to build ACRN for QEMU,
+
+   .. code-block:: none
+
+      make all BOARD_FILE=../acrn-hypervisor/misc/acrn-config/xmls/board-xmls/qemu.xml  SCENARIO_FILE=../acrn-hypervisor/misc/acrn-config/xmls/config-xmls/qemu/sdc.xml
+
+ For more details, refer to :ref:`getting-started-building`.
+
+3. Copy acrn.32.out from build/hypervisor to SOS guest /boot/ directory.
+
+4. Update the SOS kernel so as to include the virtio-blk driver. UOS (L2 guest) uses virtio-blk
+driver to mount rootfs.
+
+   a. Clone the repo, https://github.com/projectacrn/acrn-kernel.git.
+   b. cd acrn-kernel
+   c. Copy kernel_config_uefi_sos to .config
+   d. make olddefconfig
+   e. make menuconfig
+
+      .. figure:: images/acrn_qemu_4.png
+         :align: center
+
+   f. make
```

Comment:
No need to build and install the kernel modules?

Reply:
Since all the drivers are part of the kernel, I didn't need to install the modules.

Reply:
Can we add a note explaining that?